src/halmos/env.py

```diff
@@ -0,0 +1,312 @@
+# halmos/cheatcodes/env.py
+
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()  # Load environment variables from .env file
```

Comment:
this won't load from `.env_test`, right? Don't we want `.env_test` to be `.env` in `tests/regression` actually?